### PR TITLE
Update documentation and guidelines

### PR DESCRIPTION
**Pull request by hojelse**
Add and update documentation

### CODE REVIEW

**Comment by hojelse** [comment]
Please see `contributing.md` and consider if list of categories for naming convention is a good or bad addition 😄

**Comment by hojelse** [comment]
This pull request should be squashed and merged

**Comment by hojelse** [comment]
Anyone can add one self as reviewer for this :))

File: ARCHITECTURE.md [hunk 1]
```diff
@@ -10,4 +10,4 @@ This project uses Vue.js with D3.js for the front-end and C# and .NET 5 for the
 
 ## Architecture
 
-
+![architecture.png](misc-images/architecture.png)
```
Comment (pizzaluc) [inline ARCHITECTURE.md:13]: should we update this diagram before we add it or?

**Review by pizzaluc — APPROVED** [review]
Excellent work really, really good to have before our first release

File: ARCHITECTURE.md (hunk 1 above)
Comment (hojelse) [reply ARCHITECTURE.md:13]: Not sure. Do you know something is missing or should be altered. I'm not fully familiar with the acceptance criteria.

Comment (Maasdamind) [reply ARCHITECTURE.md:13]: Maybe we should update it to the specifications mentioned by David

**Review by Maasdamind — APPROVED** [review]
This is awesome! Great work 🚀

File: ARCHITECTURE.md (hunk 1 above)
Comment (pizzaluc) [reply ARCHITECTURE.md:13]: ^ @hojelse David had some small modifications, i think you were attending your exam at that point

Comment (Maasdamind) [reply ARCHITECTURE.md:13]: Yes, if I recall correctly, he mentioned we should account for storage, even though we're not using a database ATM

**Comment by Maasdamind** [comment]
@hojelse I think you can go ahead and squash it if you are ready :)

**Pull request merged**